modules/openCypher-in-gsql/pages/openCypher-in-gsql.adoc

```diff
@@ -354,12 +354,12 @@ OpenCypher does not enforce a schema, so the query:
 
 ==== Importance
```

Comment:
This is an unusual heading name.
And I don't think it should be a subsection of the example.  Because these are the "rules" for how OpenCypher in GSQL behaves, it would either be at the same or higher level than then example.